### PR TITLE
implemented grpc client for currency service

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: Api/Services/CurrencyConversion/CurrencyRateService.cs [hunk 1]
```diff
@@ -62,12 +90,15 @@ private async Task<Result<decimal>> GetCurrent(Currencies source, Currencies tar
             
             return Result.Success(decimal.Parse(content, CultureInfo.InvariantCulture));
         }
+
+        
+        private static readonly Result<decimal> SameCurrencyRateResult =  Result.Success((decimal)1);
```
Comment (AlexeyShok) [inline Api/Services/CurrencyConversion/CurrencyRateService.cs:95]: Better use Result.Success(1.0m)

**Review by AlexeyShok — APPROVED** [review]

File: Api/Services/CurrencyConversion/CurrencyRateServiceOptions.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@ namespace HappyTravel.Edo.Api.Services.CurrencyConversion
 {
     public class CurrencyRateServiceOptions
     {
-        public Uri ServiceUrl { get; set; }
         public TimeSpan CacheLifeTime { get; set; }
```
Comment (mingazhev) [inline Api/Services/CurrencyConversion/CurrencyRateServiceOptions.cs:7]: IsGrpcMaybe?

**Review by mingazhev — APPROVED** [review]
With a small comment

File: Api/Services/CurrencyConversion/CurrencyRateService.cs (hunk 1 above)
Comment (DFSko) [reply Api/Services/CurrencyConversion/CurrencyRateService.cs]: Fixed

File: Api/Services/CurrencyConversion/CurrencyRateServiceOptions.cs (hunk 2 above)
Comment (DFSko) [reply Api/Services/CurrencyConversion/CurrencyRateServiceOptions.cs:7]: Changed to enum as disqussed

**Pull request merged**